core/src/main/java/org/pentaho/platform/scheduler2/action/ActionRunner.java

```diff
@@ -74,7 +74,7 @@ public ActionRunner() {
     // empty constructor
   }
 
-    public ActionRunner( final IAction actionBean, final String actionUser, final Map<String, Serializable> params, final
+  public ActionRunner( final IAction actionBean, final String actionUser, final Map<String, Serializable> params, final
```

Comment:
Apparently, no changes needed on this file.

Reply:
fixed